src/OmniSharp/Api/TestContext/AspNet5TestCommandProvider.cs

```diff
@@ -0,0 +1,47 @@
+using System.IO;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace OmniSharp
+{
+    public class AspNet5TestCommandProvider : ITestCommandProvider
+    {
+        public string GetTestCommand(TestContext testContext)
+        {
+            if (new FileInfo(testContext.ProjectFile).Extension != ".json")
```

Comment:
Yes, we should be using the projects that omnisharp is aware of. We need to expose the AspNet5Context here so we can see if the project file is one we're aware of:

https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp/AspNet5/AspNet5Context.cs